apps/frontend/src/store/server.ts

```diff
@@ -35,6 +35,8 @@ export interface IServerState {
   ldap: boolean;
   localLoginEnabled: boolean;
   userInfo: IUser;
+  tenableHostUrl: string | undefined;
+  splunkHostUrl: string | undefined;
```

Comment:
```suggestion
  tenableHostUrl: string;
  splunkHostUrl: string;
```

those urls will always be strings right?